CHANGELOG.md

```diff
@@ -68,7 +68,7 @@
   the CA in AWS.
 
   The implementation supports multiple CAs. For configuration information, see [the
-  certificate authority settings section in the docs](https://lyft.github.io/confidant/configuration.html#certificate-authority-settings).
+  certificate authority settings section in the docs](configuration.html#certificate-authority-settings).
```

Comment:
Err am I missing something? These links don't work for me - even when I use master https://github.com/lyft/confidant/blob/master/CHANGELOG.md